docs/content/TextInput.mdx

```diff
@@ -81,6 +81,12 @@ TextInput is a form component to add default styling to the native text input.
 <TextInput contrast aria-label="Zipcode" name="zipcode" placeholder="Find user" autoComplete="postal-code" />
 ```
 
+## Monospace text input
+
+```jsx live
+<TextInput monospace aria-label="Zipcode" name="zipcode" placeholder="Find user" autoComplete="postal-code" />
```

Comment:
Although it's just an example, shall we switch to using personal access token instead? I don't think we'd ever want post code in monospace 🤔 , so best to avoid giving that impression.

```suggestion
<TextInput monospace aria-label="personal access token" name="pat" placeholder="github_pat_abcdefg123456789" />
```

Reply:
Ah thanks! This is helpful!